packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-customer-strategy.ts

```diff
@@ -242,7 +242,11 @@ export default class PayPalCommerceCustomerStrategy implements CustomerStrategy
             await this.paymentIntegrationService.selectShippingOption(shippingOption.id);
             await this.paypalCommerceIntegrationService.updateOrder();
         } catch (error) {
-            throw new Error(error);
+            if (typeof error === 'string') {
+                throw new Error(error);
+            }
+
+            throw error;
```

Comment:
Lets change it with `this.handleError(error)`

Reply:
Same as above.
As this type check is definitely working, I think we don't need to combine `throw new Error(error)` into the exsiting `private handleError(error: unknown) {` logic.

Reply:
Same as above.
I think we have to update catch implementation with this.handleError(error) because it gives an ability to handle error through onError option provided from UI